.github/workflows/up_ex_doc_version.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: up_ex_doc_version
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run daily at midnight
+  workflow_dispatch:
+  
+
+jobs:
+  test:
+    name: Update ex_doc version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Get current version
+        run: |
+            CURR=`curl -s https://hex.pm/api/packages/ex_doc | jq -r '.latest_stable_version' \
+                  jq -r ".[0].name"
```

Comment:
```suggestion
            CURR=$(curl -s https://hex.pm/api/packages/ex_doc | jq -r '.latest_stable_version')
```

Reply:
This command was incomplete (no closing "`"). Maybe it was a suggestion of mine, but here it's fixed.